reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Jobs.md

```diff
@@ -1,371 +1,418 @@
+  maintained on the originating computer.
 
-### START A BACKGROUND JOB IN AN INTERACTIVE SESSION
+### Start a background job in an interactive session
 
 You can start an interactive session with a remote computer and then start a
 background job during the interactive session. For more information about
-interactive sessions, see about_Remote, and see Enter-PSSession.
+interactive sessions, see about_Remote, and see `Enter-PSSession`.
 
-The procedure for starting a background job in an interactive session is
-almost identical to the procedure for starting a background job on the local
-computer. However, all of the operations occur on the remote computer, not the
-local computer.
-
-#### STEP 1: ENTER-PSSESSION
-
-Use the Enter-PSSession cmdlet to start an interactive session with a remote
-computer. You can use the ComputerName parameter of Enter-PSSession to
-establish a temporary connection for the interactive session. Or, you can use
-the Session parameter to run the interactive session in a Windows PowerShell
-session (PSSession).
-
-The following command starts an interactive session on the Server01 computer.
-
-```powershell
-C:\PS> Enter-PSSession -computername Server01
-```
-
-The command prompt changes to show that you are now connected to the Server01
+The procedure for starting a background job in an interactive session is almost
+identical to the procedure for starting a background job on the local computer.
+However, all of the operations occur on the remote computer, not the local
 computer.
 
-```
-Server01\C:>
-```
 
-#### STEP 2: START-JOB
+1. Use the `Enter-PSSession` cmdlet to start an interactive session with a
+   remote computer. You can use the ComputerName parameter of `Enter-PSSession`
+   to establish a temporary connection for the interactive session. Or, you can
+   use the Session parameter to run the interactive session in a PowerShell
+   session (PSSession).
 
-To start a background job in the session, use the Start-Job cmdlet.
+   The following command starts an interactive session on the Server01 computer.
 
-The following command runs a background job that gets the events in the
-Windows PowerShell event log on the Server01 computer. The Start-Job cmdlet
-returns an object that represents the job.
+   ```powershell
+   C:\PS> Enter-PSSession -computername Server01
+   ```
 
-This command saves the job object in the \$job variable.
+   The command prompt changes to show that you are now connected to the
+   Server01 computer.
 
-```powershell
-Server01\C:> $job = start-job -scriptblock {
-  get-eventlog "Windows PowerShell"
-}
-```
+   ```
+   Server01\C:>
+   ```
 
-While the job runs, you can use the interactive session to run other commands,
-including other background jobs. However, you must keep the interactive
-session open until the job is completed. If you end the session, the job is
-interrupted, and the results are lost.
+1. To start a background job in the session, use the `Start-Job` cmdlet. The
```

Comment:
I wonder if we should remove this, since 'about_jobs' talks about background jobs and this is supposed to be about remote jobs.

Reply:
Fixed